roles/neutron-common/templates/etc/neutron/metadata_agent.ini

```diff
@@ -1,7 +1,7 @@
 [DEFAULT]
 debug = False
 
-auth_url = https://{{ endpoints.keystone }}:35358/v2.0
+auth_url = {{ endpoints.keystone.url.admin }}/{{ endpoints.keystone.version  }}
```

Comment:
Looks like an extra space between `version` and `}}`. Should be a max of one space.